docs/data.ts

```diff
@@ -11,11 +11,11 @@ export const guideLinks = [
 ];
 
 export const tutorialLinks = [
-  { text: '1. Building your first WebContainer app', link: '/tutorial/1-building-your-first-webcontainers-app' },
-  { text: '2. Setting up WebContainers', link: '/tutorial/2-setting-up-webcontainers' },
-  { text: '3. Installing dependencies', link: '/tutorial/3-installing-dependencies' },
-  { text: '4. Running dev server', link: '/tutorial/4-running-dev-server' },
-  { text: '5. Editing & saving a file', link: '/tutorial/5-editing-a-file-updating-the-iframe' },
+  { text: '1. Build your first WebContainer app', link: '/tutorial/1-building-your-first-webcontainers-app' },
```

Comment:
I like this! 

Though, we need to match the sidebar title with the URL and the page title (feedback I received from numerous readers + SEO/accessibility implications). Do you wanna change it or should we have that as another PR?

Reply:
I'll change it.
I'll also increase the Sidebar width a bit more then, to accommodate the longer title.
It looks bad when there's a line break only for the last word:
```
1. Building your first WebContainer
app
```

Reply:
One line, even with scrollbar:

<img width="321" alt="Screen Shot 2023-02-21 at 5 17 02 PM" src="https://user-images.githubusercontent.com/22125230/220488171-5b161988-460e-4846-82c7-c9a18c62fe6b.png">

Reply:
If the page title is the "source of truth", perhaps it would be good in the future to cater a bit to the nav link text length when naming the page title, so the link titles are easier to scan and are more likely to fit on one line.

Reply:
Oh I meant - if you'd change the titles/file names not if you'd revert back ✨ I'm sorry for being unclear.